src/gvpn_controller.py

```diff
@@ -1,22 +1,23 @@
 #!/usr/bin/env python
 
 from ipoplib import *
-
+# GvpnUdpServer inherits the UdpServer class of the core IPOP Library
 class GvpnUdpServer(UdpServer):
     def __init__(self, user, password, host, ip4):
         UdpServer.__init__(self, user, password, host, ip4)
         self.idle_peers = {}
-        self.user = user
-        self.password = password
-        self.host = host
+        self.user = user # XMPP username
+        self.password = password # XMPP Password
+        self.host = host # XMPP host
         self.ip4 = ip4
-        self.uid = gen_uid(ip4)
+        self.uid = gen_uid(ip4) # UID is the SHA-1 hash of the IPv4 address.
         self.vpn_type = "GroupVPN"
         self.ctrl_conn_init()
 
         self.uid_ip_table = {}
         parts = CONFIG["ip4"].split(".")
         ip_prefix = parts[0] + "." + parts[1] + "."
+        # Popluating the uid_ip_table with all the IPv4 addresses and the corresponding UIDs in the /16 subnet.
```

Comment:
s/Popluating/Populating/

Also, wrap comments to 80 columns? 79 columns? Are we following PEP-8? @pstjuste
The same thing goes for other places you go over.